content/concept-transaction-cost.md

```diff
@@ -57,17 +57,20 @@ See also: [Fee Escalation explanation in `rippled` repository](https://github.co
 
 ### Queued Transactions ###
 
-(Requires the [FeeEscalation amendment](concept-amendments.html#feeescalation))
-
 When `rippled` receives a transaction that meet the server's local load cost but not the open ledger cost, the server estimates whether the transaction is "likely to be included" in a later ledger. If so, the server adds the transaction to the transaction queue and relays the transaction to other members of the network. Otherwise, the server discards the transaction. The server tries minimize the amount of network load caused by transactions that would not pay a transaction cost, since [the transaction cost only applies when a transaction is included in a validated ledger](#transaction-costs-and-failed-transactions).
```

Comment:
"The server tries minimize the amount of network load caused by transactions that would not pay a transaction cost," --> missing "to"